spec/operations/project_operations_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
```

Comment:
A tip, to make even easier to maintain would be to separe a new spec file for each kind of operation, otherwise this can become a very huge and ugly file. (which is not the case here, but i just thought it would bring more good than harm 😄 )